### ISSUE TITLE
[0.9] Add mechanical components to workstations and vehicles

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoSuggestions#1077]
A bit related to #1044 .

Basically, at the moment there is a huge first mover advantage to being the first person on the server that makes Electronics, Industry, Machinist, etc. items. Everyone wants a Truck, but once they have a Truck, almost nobody wants another Truck. If you get the skills to make those items everyone wants one of a week later than some people, you don't get much use of them.

Similarly, everyone seems to supply their own intermediate products (axels, wheels, engines, etc.) rather than buying / selling them on the market.

It would perhaps be nice if many of those products came with components that degrade over time. A Truck might have an axel component that wears down and you need to replace it with another axel to keep going. Same with tires, engines, etc. This way there would be a constant need for parts suppliers - even if you can't make a Truck because you don't have Electronics, you could still sell axels and be in business. This would also encourage more of those parts being on the market, hopefully encouraging the high-end manufacturers to purchase some of those parts from the new manufacturers to save themselves time and effort (relating a bit to #1073).

You could have the same mechanics at play for workstations - a wind turbine would require the motor to be replaced every now and then, a pumpjack would require some iron components, an anvil would require a bit of raw iron to get touched up, etc.

On the flipside, this would either require those parts to have durability, making them impossible to stack (which would be awful), or they would have to be consumed like fuel (also not ideal, especially when you're trying to resell that item).

**Comment by trickenso** [comment]
> On the flipside, this would [] require those parts to have durability, making them impossible to stack (which would be awful)

Agreed. This mightn't be a problem if the part could never be removed after installation - I'm not sure why I would have several axles in storage in various states of disrepair, for instance.

Agreed that this complicates resale, though. Something like the 'only at full repair' store option should be able cover this, though, right?

Another vote from me for increasing ongoing input to the economy.

**Comment by Sarah-Graesyn** [comment]
hmmm, I like this idea actually. Like... some good thoughts here.
